template/src/core/network/globalNetworkModels.d.ts

```diff
@@ -0,0 +1,19 @@
+interface ListRequestResponse<Result> {
+  count: number;
+  next: null | string;
+  previous: null | string;
+  results: Result[];
+}
+
+type ListRequestParams<Ordering = string> = Partial<{
+  ordering: Ordering;
+  limit: "all" | number;
+  offset: number;
+  search: string;
+}>;
+
+interface HipoApiErrorShape {
+  type: ArrayToUnion<typeof ERROR_TYPES>;

```

Comment:
As far as I can see, `ERROR_TYPES` is never defined anywhere. When we are adding something to the template, it should be fully integrated. We can define those somewhere but may be it is better to simply type this as `string`. Not sure though so we can discuss it together.